src/models/Teams.php

```diff
@@ -138,9 +138,10 @@ public function create(string $name): int
         $name = Filter::sanitize($name);
 
         // add to the teams table
-        $sql = 'INSERT INTO teams (name, link_name, link_href) VALUES (:name, :link_name, :link_href)';
+        $sql = 'INSERT INTO teams (name, common_template, link_name, link_href) VALUES (:name, :common_template, :link_name, :link_href)';
         $req = $this->Db->prepare($sql);
         $req->bindParam(':name', $name);
+        $req->bindParam(':common_template', Templates::defaultBody);
```

Comment:
I think it should be bindValue here.

See: Fatal error: Uncaught Error: PDOStatement::bindParam(): Argument #2 ($var) cannot be passed by reference in /home/scrutinizer/buil
/src/models/Teams.php on line 144

![2021-03-03-124138_953x414_scrot](https://user-images.githubusercontent.com/3043706/109800709-d555d500-7c1d-11eb-96ce-cf9e532baa3b.png)

Reply:
Thanks for the suggested changed. Fixed in cfdd838